src/applications/public-outreach-materials/libraries/library-filters.js

```diff
@@ -36,9 +42,18 @@ export function libraryCount() {
 }
 
 export function libraryCurrent() {
+  let increment = 1;
+  const itemsPerPage = 10;
+  let numVal;
   cards.forEach(element => {
-    const numVal = element.getAttribute('data-number');
-    const itemsPerPage = 10;
+    if (
+      !element.classList.contains('hide-topic') &&
+      !element.classList.contains('hide-type')
+    ) {
+      element.setAttribute('data-number', increment);
+      numVal = element.getAttribute('data-number');
```

Comment:
I don't think this would be any more concise, and seems harder to read in this scenario, @cvalarida. Apologies if I'm missing what you are trying to convey.

Reply:
The way this function is written right now, we don't update `numVal` in some cases. If I'm understanding this all correctly, this causes the next `if` statement to do what it did last time to the new element whether it should or not.

Consider what happens when we have the following cards:
```javascript
const cards = [
  // 9 cards omitted for clarity
  { classList: [] }, // index: 9
  { classList: ['hide-topic'] } // index: 10
]
```
When we get to the 9th index (tenth item—what should be the last card on the page, right?), we'll get:
```javascript
if (
  !element.classList.contains('hide-topic') && // true
  !element.classList.contains('hide-type') // true
) {
  numVal = 9
}
if (
  9 > 1 * 10 || // false
  9 <= (1 - 1) * 10 // false
) {
  // skipped
} else {
  // `pager-hide` class removed
}
```
When we move on to the next card—the first one that should be hidden because it's out of the range of the active page—we get:
```javascript
if (
  !element.classList.contains('hide-topic') && // false
  !element.classList.contains('hide-type') // true
) {
  // skipped; numVal is still 9
}
if (
  9 > 1 * 10 || // false
  9 <= (1 - 1) * 10 // false
) {
  // skipped
} else {
  // `pager-hide` class removed
}
```

It will show all `hide-topic` or `hide-type` cards after the page limit until we hit a non-`hide-topic or `hide-type` card.

What I'm proposing is keeping it basically like it was before, but adding and additional check to the original `if` statement that only removes the `pager-hide` class if the card shouldn't be hidden per one of those `hide-*` classes.

Would that still accomplish what you're trying to do?

Reply:
Apologies - still not with you on this, @cvalarida - trying to restructure this function per your suggestion breaks my pager. Can you give me a verbatim example of how you think my function should look, and I can try plugging it in?

Reply:
@cvalarida Cards have a numVal by default that is set on the backend `element.getAttribute('data-number')` - we don't have to update it. If it's not updated, that means no filtering has occurred.

Reply:
Update: @ethanteague and I discussed this in Slack. Gonna be honest: I have no idea how this is working, but according to the screenshot he showed, it looks fine. :man_shrugging: